azure-pipelines.yml

```diff
@@ -183,6 +172,11 @@ jobs:
       inlineScript: 'az aks get-credentials -n gpitfutures-dev-aks -g gpitfutures-dev-rg-aks'
     name: kubectlContext
     displayName: Get Kubernetes Context
+
+  - bash: 'bash determine-namespace.sh'
```

Comment:
running it with shell `./foo` gave me [Permission denied](https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_build/results?buildId=8313&view=logs&jobId=2d068747-3436-5bb0-8267-bf12ae7ef269&j=2d068747-3436-5bb0-8267-bf12ae7ef269&t=160d3d53-7022-5856-ff9d-d4f560ca7968)
I suspect it is because the file  I checked in doesn't have x permission. 
hence why I'm running it explicitly with bash as that doesn't execute the file, just feeds it into `/bin/bash`.

Reply:
git update-index --chmod=+x
you can tell git explicitly something is executable